src/routes/questionnaires.js

```diff
@@ -56,6 +56,8 @@ exports.plugin = {
           query: Joi.object({
             course: Joi.objectId(),
             program: Joi.objectId(),
+            // [temporary] This line can be removed when mobile versions prior to 2.25.0 have been deprecated.
```

Comment:
J'ai noté de retirer ce morceau de code une fois qu'on aura fait la MEP mobile : https://www.notion.so/compani/Distribution-mobile-MEP-d7238605d8c74cc59fa724ca5d94f253?pvs=4#44e6261e77a0404f90a12a5a66261d47